charts/puris-frontend/values.yaml

```diff
@@ -0,0 +1,106 @@
+# Default values for puris-frontend.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: ghcr.io/catenax-ng/tx-puris-frontend/puris-frontend
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  # tag: "latest"
```

Comment:
Provide image tag as empty key `tag: ""` instead of commenting it out.

Reply:
Applied the changes as suggested!